docs/make.jl

```diff
@@ -29,9 +29,13 @@ makedocs(sitename="Breeze",
     ]
 )
 
-deploydocs(;
+deploydocs(
+    ;
     repo = "github.com/NumericalEarth/Breeze.jl",
+    deploy_repo = "github.com/NumericalEarth/BreezeDocumentation",
     devbranch = "main",
-    push_preview = true,
-    forcepush = true
+    # Only push previews if all the relevant environment variables are non-empty. This is an
+    # attempt to work around https://github.com/JuliaDocs/Documenter.jl/issues/2048.
+    push_preview = all(!isempty, (get(ENV, "GITHUB_TOKEN", ""), get(ENV, "DOCUMENTER_KEY", ""))),
```

Comment:
I'd need to wait for a PR by dependabot to be sure this is working correctly, but looking at https://github.com/NumericalEarth/Breeze.jl/actions/runs/19032077021/job/54348250773?pr=61#step:5:5 `GITHUB_TOKEN` isn't empty, but `DOCUMENTER_KEY` is, so making sure they're both non-empty should be a decent approach.